recipes/jalview/meta.yaml

```diff
@@ -1,25 +1,25 @@
 package:
   name: jalview
-  version: 2.10.5
+  version: 2.11.0
 
 build:
+  number: 0
   noarch: generic
-  number: 3
 
 source:
-  url: http://www.jalview.org/source/jalview_2_10_5.tar.gz
-  sha256: "def6eed9bdc4f43efc020717f283f3591da113f36f6bb18718339951f6b1707f"
+  url: http://www.jalview.org/source/jalview_2_11_0.tar.gz
+  sha256: "18fd7d6377a46bf7a45a9c1f6ee17a3cffe0ed429555ba5bb713c1ad1e59da7f"
 
 requirements:
   build:
-    - openjdk >8.0.121
-    - ant
+    # jalview 2.11.0 requires Java 11 or greater to build.
+    - gradle >5.0
+    - openjdk 11
     - xorg-libxtst
   run:
-    # Enforce a version requirement on openjdk to ensure
-    # it comes from the conda-forge channel and not default.
+    # Jalview 2.11.0 is 1.8 only for execution
     # Version number reference: https://github.com/conda/conda/issues/6948#issuecomment-369360906
-    - openjdk >8.0.121
+    - openjdk 8.0.192
     - xorg-libxtst
     - psutil
```

Comment:
`openjdk 11` will work fine, you don't need any equals signs.

Reply:
OK, my bad.
And regarding `openjdk 8.0.192`, what's the meaning for the conda warning about adding `.*` to version pinnings and using operators?

```
CONDA_BUILD WARNING Adding .* to spec 'openjdk 8.0.192' to ensure satisfiability.  Please consider putting {{ var_name }}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml
```

Reply:
It's just mentioning that it's adding a wildcard in so that `8.0.192` will match patch versions. Anytime you pin to a version conda will add a wildcard onto that to add some flexibility when solving.